### DISCUSSION TITLE
Change domain

### DISCUSSION THREAD

**Discussion by mquandalle** [plausible/analytics#1472] [Anything Else]
Hi,

I would like to change the domain of my website but it seems that the feature is disabled in the settings UI ?

**Comment by metmarkosaric** [comment]
instructions on how to change the domain name: https://plausible.io/docs/change-domain-name

**Reply by mquandalle** [reply]
Ok thank you for the detailed informations. I'm fine with keeping the old domain reference in the script, the only minor downside is that my new domain isn't reflected in Plausible UI.
I'll migrate when the option is available :)

**Reply by metmarkosaric** [reply]
sounds like a plan, thanks!
